src/components/Media/Media.tsx

```diff
@@ -34,7 +34,7 @@ export function Media({ className, image, style, title }: Props) {
                 playsInline
                 style={style}
                 title={title}
-                webkit-playsinline
+                webkit-playsinline="true"
```

Comment:
React was complaining about boolean value being used for a non-boolean attribute.